src/Nethermind/Nethermind.AuRa/AuRaBlockProcessor.cs

```diff
@@ -0,0 +1,63 @@
+﻿//  Copyright (c) 2018 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections.Generic;
+using Nethermind.Blockchain;
+using Nethermind.Blockchain.Receipts;
+using Nethermind.Blockchain.Rewards;
+using Nethermind.Blockchain.TxPools;
+using Nethermind.Blockchain.Validators;
+using Nethermind.Core;
+using Nethermind.Core.Specs;
+using Nethermind.Evm;
+using Nethermind.Evm.Tracing;
+using Nethermind.Logging;
+using Nethermind.Store;
+
+namespace Nethermind.AuRa
+{
+    public class AuRaBlockProcessor : BlockProcessor
+    {
```

Comment:
Seems confusing at first glance that AuRaBlockProcessor does not implement IAuRaBlockProcessor

Reply:
Renamed to IAuRaBlockProcessorExtension